### ISSUE TITLE
7thweb.net down ??

### ISSUE THREAD

**Issue by chadouhoney** [sebfz1/wicket-jquery-ui#355]
Not much of a project issue, but I didn't know where to address this. Since the last 2-3 weeks I do not have access to the site. Is there some maintenance going on, has the address changed, or what?

**Comment by sebfz1** [comment]
Hi, yes unfortunately the server's HD has crashed and I didn't had time to
setup a new one. Sorry for the inconvenience, it might take another week or
two to find some time to dedicate to this. :(

**Comment by chadouhoney** [comment]
Any updates on this one?

**Comment by Patrick1701** [comment]
@chadouhoney The project has been moved to Wicket Stuff. Sebastien does not maintain here anymore.

**Comment by sebfz1** [comment]
Yes, I'm so sorry I couldn't maintain the project anymore. My new life (in
Japan) has been very busy from the start, leaving me absolutely no room for
spare development. I'm very happy with this new life and what I'm doing
here, but a little sad I had to be separated from this project I cared
about so much. But... that's life! Thank you for your understanding! 🙏
